external/vcm/vcm/catalog.yaml

```diff
@@ -7,7 +7,7 @@ sources:
     driver: zarr
     args:
       storage_options:
-        project: 'vcm-ml'
+
```

Comment:
Did you mean to leave an empty line in all of these?

Reply:
Fixed. This was a side-effect of find-and-replace.